### ISSUE TITLE
Add a dropdown menu for selecting the strain on the genotype creation page

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1421]
From https://github.com/PHI-base/canto/issues/14

I've closed that issue.  There have been code changes due to #1381 that make quite a bit of that discussion out of date.

------------------------

The drop-down will be pre-populated using the organism to strain mapping.

Thinking ahead, we'll need to be able to select the host strain and the pathogen strain separately.

Notes from meeting on Jan 15th 2018:

  -  A drop down list would probably be OK as the number of strains is small-ish
  - Later, there will need to be an "Other" option that brings up a free-text box for saving a new strain in Canto

------------------------

Now there is a test web service available (https://github.com/PHI-base/canto/issues/20), the drop down menu can be populated using that.

In GenotypeManageCtrl, the service with something like this (untested):

```javascript
  CantoService.lookup('strains', [taxonid])
      .then(function(response) {
            var strains = response.data;
            ....
      }); 
```

**Comment by kimrutherford** [comment]
> In GenotypeManageCtrl, the service with something like this (untested):

Now that #1420 is done the strain lookup web service will now return real data.

@barnabynorman @jseager7 when you get a chance, could you try the web service using the example code above?

**Comment by kimrutherford** [comment]
Great.  Thanks for testing.

The next steps are:

 - show a strains dropdown for an organism only once an organism is selected on the genotype management page
 - show two dropdowns, one to host and one for pathogen
 - wire up the strain dropdowns to update if a different organism is selected

I'm happy to chat about any of that next week.

**Comment by kimrutherford** [comment]
I think this is superseded by #1610
